zeppelin-server/src/main/java/org/apache/zeppelin/rest/NotebookRestApi.java

```diff
@@ -14,13 +14,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package org.apache.zeppelin.rest;
 
 import com.google.common.collect.Sets;
 import com.google.common.reflect.TypeToken;
 import com.google.gson.Gson;
 
 import org.apache.commons.lang3.StringUtils;
+import org.apache.zeppelin.rest.message.CronRequest;
```

Comment:
Can you check the order of dependencies? It seems to be wrong

Reply:
btw, @jongyoul how about update [_tools/checkstyle.xml](https://github.com/apache/zeppelin/blob/master/_tools/checkstyle.xml) to automatically check import order?

Reply:
I think it would be better, actually.

Reply:
Personally, I'm okay with any format / criteria that checkstyle plugin can forces.